project-icp/bee-pollinator-app
# Improve CDL Picker UI

## Overview

Makes the CDL picker more noticeable, in accordance with mockups.

Connects #485 

### Demo

Tagging @jfrankl for visual / style review.

<img width="1236" alt="image" src="https://user-images.githubusercontent.com/1430060/54157439-b3eef380-441e-11e9-852c-4dcc23f47613.png">

<img width="1236" alt="image" src="https://user-images.githubusercontent.com/1430060/54157461-bf421f00-441e-11e9-96a5-db0d22b9217d.png">

## Testing Instructions

* Check out this branch and `beekeepers start`
* Go to [:8000/?beekeepers](http://localhost:8000/?beekeepers) and interact with the CDL selector
